api/src/opentrons/protocol_api/core/engine/instrument.py

```diff
@@ -1493,7 +1755,24 @@ def aspirate_liquid_class(
                 volume=volume,
             )
         components_executor.aspirate_and_wait(volume=volume)
-        components_executor.retract_after_aspiration(volume=volume)
+        if (
+            transfer_type == tx_comps_executor.TransferType.ONE_TO_MANY
+            and conditioning_volume not in [None, 0]
```

Comment:
```suggestion
            and not conditioning_volume
```

I'm not always a fan of relying on `not 0` or `not None` to be used as booleans, but since both are valid and since I think it reads fine, I'd prefer this. Not a strong opinion though

Reply:
We want to check the opposite though.. that conditioning volume is a non-zero number. The alternative could be to just use `.. and conditioning_volume` or `.. and conditioning_volume is not None and conditioning_volume > 0`. The first one is concise but loses the detail that we are looking for a non-zero number, and the second one is too verbose. I think the `..not in [None, 0]` is a good middle-ground.